protocol/src/main/java/io/camunda/zeebe/protocol/record/ImmutableProtocol.java

```diff
@@ -73,7 +73,9 @@
     code = "(builder=[[builder]])") // mark generated class as part of immutable protocol
 public @interface ImmutableProtocol {
 
-  /** @return the builder class for this abstract protocol type */
+  /**
+   * @return the builder class for this abstract protocol type
+   */
```

Comment:
🔧 Please change the return tag into a summary fragment:

```java
/** Returns the builder class for this abstract protocol type */
```

See https://github.com/camunda/zeebe/issues/9045